components/brave_wallet_ui/components/extension/popup-modals/index.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+
+// Styled Components
+import {
+  StyledWrapper,
+  Header,
+  Title,
+  CloseButton,
+  Modal
+} from './style'
+
+export interface Props {
+  children?: React.ReactNode
+  onClose: () => void
+  title: string
+}
+
+const ESC_KEY = 'Escape'
+
+const PopupModal = (props: Props) => {
```

Comment:
allow non-default export

Reply:
Do you suggest both default as well as non-default export here like this?

```
export { PopupModal }
export default PopupModal
```

Reply:
non-default export via: `export const PopupModal = ...`. 

default export optional

Reply:
Also we should move towards naming all files uniquely, instead of all component files named `index.tsx`.

suggesting rename to `pop-up-modals.tsx` & `pop-up-modals.style.tsx`